### ISSUE TITLE
Spain - Remove April 23 from Barcelona

### ISSUE THREAD

**Issue by florin-bujoreanu** [nager/Nager.Date#403]
### Affected country

Spain

### Affected public holiday

Dia de Castilla y Leon

### Source of the information

https://www.officeholidays.com/countries/spain/catalonia/2022

https://www.boe.es/diario_boe/txt.php?id=BOE-A-2021-17113

**Comment by tinohager** [comment]
Is there an official source for the adjustments, wikipedia, the government?

**Comment by florin-bujoreanu** [comment]
I could not find a wikipedia article on this regard.
The list has been confirmed by our organization users who reported the holidays for which I raised the issues.

**Comment by fidelpita** [comment]
I don't think there is any problem in this case.
The issue title is "Remove April 23 from Barcelona" but April 23rd is **not** being presented as a public holiday in Catalonia (region where Barcelona is located). For information, April 23rd is an important day in Catalonia (Sant Jordi) but is not a public holiday. April 23rd is **correctly** presented as a public holiday in the region of Aragón with the name of "San Jorge (Día de Aragón)" and also **correctly** for the region of Castilla y León with the name of "Día de Castilla y León". 

Official source: https://www.boe.es/diario_boe/txt.php?id=BOE-A-2021-17113
Nager.date: https://date.nager.at/PublicHoliday/Country/ES

In summary:
- Barcelona is located in the region of Catalonia (Cataluña)
- April 23rd is not presented by nager.date as a public holiday in Cataluña -> **Correct**
- April 23rd is presented as a public holiday in the region of Aragón with the name of "San Jorge (Día de Aragón)" -> **Correct**
- April 23rd is presented as a public holiday in the region of Castilla y León with the name of "Día de Castilla y León" -> **Correct**

**Comment by florin-bujoreanu** [comment]
Please apologize for the incorrect ticket, there has been a misunderstanding from my side, I interpreted CL as CT.

**Comment by tinohager** [comment]
@fidelpita **Thank you so much for your great support.** In the last few days, countless new issues have been recorded for Spain. I would be happy if you could contribute your great expertise there as well.

**Comment by fidelpita** [comment]
@tinohager Glad to be of help. I will try to have a look at those issues you mentioned.
